test/coverage-test/custom-provider.ts

```diff
@@ -11,6 +11,10 @@ const CustomCoverageProviderModule: CoverageProviderModule = {
   takeCoverage() {
     return { customCoverage: 'Coverage report passed from workers to main thread' }
   },
+
+  // TODO: Figure out how to verify these were called in tests. Maybe write file to "./coverage" ?
+  startCoverage() {},
+  stopCoverage() {},
```

Comment:
I verified manually that these are indeed called during tests. I'll need to think proper way to include these in tests.